src/lib/components/atoms/ApiSettingsButton.svelte

```diff
@@ -0,0 +1,123 @@
+<script lang="ts">
+	import * as settings from '../../../data/api/settings';
+	import { clear } from '../../../data/api/cache';
+	let url = settings.api_base_url();
+	function handler() {
```

Comment:
> This is never going to work now because we forgot to reorder this after making `clear()` reload. Speaking of that, the `clear()` should probably also be renamed to `clear_and_reload()` or something.

Ugh I was referring to the `reset()` function. GitHub UI moment